charts/spegel/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: spegel
 description: Stateless cluster local OCI registry mirror.
 type: application
-version: v0.0.1
-appVersion: v0.0.1
+version: v0.0.13
+appVersion: v0.0.13
```

Comment:
Do not set the versions. They are set during build time along with other variables.

Reply:
made the changes in #220 PR